mmdet/models/dense_heads/yolo_head.py

```diff
@@ -257,20 +257,23 @@ def _get_bboxes_single(self,
             # (h, w, num_anchors*num_attrib) -> (h*w*num_anchors, num_attrib)
             pred_map = pred_map.permute(1, 2, 0).reshape(-1, self.num_attrib)
 
-            pred_map[..., :2] = torch.sigmoid(pred_map[..., :2])
-            bbox_pred = self.bbox_coder.decode(multi_lvl_anchors[i],
-                                               pred_map[..., :4], stride)
+            xy = torch.sigmoid(pred_map[..., :2])
```

Comment:
Is this really necessary to work around this way? The old version works fine with me when converting to ONNX.

Reply:
Yeah, this is the most critical part that TensorRT didn't like. I think the reason is that it's an inplace-write on a slice, which doesn't seem to be supported (agreed, it works in ONNX/onnx-runtime though)